### PR TITLE
Add krishna to people page

### PR DESCRIPTION
**Pull request by krishnachaitanya9**
Kindly add me to the people page.

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -46,6 +46,13 @@
    email: chiju.wu@colorado.edu
    pic: sayter.jpg
 
+- name: Krishna Chaitanya Kodur
```
Comment (joewie) [inline _data/people.yml]: The indentation is wrong here.

Comment (alecive) [reply _data/people.yml]: Good catch, this would break everything

**Comment by peasant98** [comment]
indention is fixed; should render correctly on deploy preview.

File: _data/people.yml [hunk 2]
```diff
@@ -46,6 +46,13 @@
    email: chiju.wu@colorado.edu
    pic: sayter.jpg
 
+ - name: Krishna Chaitanya Kodur
+   role: grad
+   url: https://www.linkedin.com/in/kodur-krishna-chaitanya/
```
Comment (joewie) [inline _data/people.yml]: Your LinkedIn profile isn't actually visible at this link.

Comment (joewie) [reply _data/people.yml]: Looks good now!

**Comment by krishnachaitanya9** [comment]
Thanks, everyone!

**Pull request closed without merging**